front/screens/ListArticles.tsx

```diff
@@ -0,0 +1,99 @@
+import type { StackNavigationProp } from "@react-navigation/stack";
+import type { FC } from "react";
+import * as React from "react";
+import { StyleSheet, Text } from "react-native";
+import { Image, ListItem } from "react-native-elements";
+
+import { View } from "../components/Themed";
+import Colors from "../constants/Colors";
+import type { TabHomeParamList } from "../types";
+
+interface Props {
+  navigation: StackNavigationProp<TabHomeParamList, "listArticles">;
+}
+
+const ListArticles: FC<Props> = ({ navigation }) => {
+  const screenTitle = "Nom de l'étape";
+  const description =
+    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.";
+  const list = [
```

Comment:
```suggestion
  const articles = [
```
Avec le `[]`, on voit que c'est une liste mais on veut savoir ce qu'elle contient

Reply:
Ah oui, j'ai laissé la variable de l'exemple du ListItem (React Native Element)